src/include/miopen/solver.hpp

```diff
@@ -4196,6 +4196,59 @@ struct ConvAsmImplicitGemmGTCDynamicFwdDlopsNCHWC final
                 const PerformanceConfigAsmImplicitGemmGTCFwdDlopsNCHWC& config) const override;
 };
 
+struct PerformanceConfigHipImplicitGemmFwdXdlops
+    : Serializable<PerformanceConfigHipImplicitGemmFwdXdlops>
```

Comment:
Why based on Serializable? should be based on PerfConfig

Reply:
For classes based on PerfConfig, we have defined output operator. But for Serializable we don't.

Reply:
This was a merge error since Evgenii's PR was merged in the interim, I have fixed the error.

Reply:
oh, that explains why test failed only in develop.